src/core/buffer/representation_buffer.ts

```diff
@@ -319,6 +327,23 @@ export default function RepresentationBuffer<T>({
       }));
   }
 
+  /**
+   * Returns stream events.
+   * @param data
+   */
+  function handleInbandStreamEvent(
+    value: ISegmentObject<T>
+  ): Observable<IBufferInbandStreamEvent> {
+    const { segmentEvent } = value;
+    if (segmentEvent) {
+      return observableOf({
+        type: "inband-stream-event" as "inband-stream-event",
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Remove this unnecessary cast. ([typescript:S4325](https://sonarcloud.io/organizations/rx-player/rules#rule_key=typescript%3AS4325))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=rx-player&pullRequest=247&issues=AWSyshg0ummypApIUp1q&open=AWSyshg0ummypApIUp1q)

[ISSUE_KEY]: AWSyshg0ummypApIUp1q